WebJobs.Script.sln

```diff
@@ -160,6 +160,49 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "BlobTrigger-Batch", "BlobTr
 		sample\BlobTrigger-Batch\run.bat = sample\BlobTrigger-Batch\run.bat
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "EasyTables", "EasyTables", "{3EACC3F3-2747-42D6-B826-6A150F9EA92A}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CSharp", "CSharp", "{9A295BFF-005D-456B-BBEA-6F4F389B7A66}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\host.json = sample\EasyTables\CSharp\host.json
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "QueueToOutItem", "QueueToOutItem", "{5DF1393D-B53A-47C3-895D-A7F3071F44F1}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\QueueToOutItem\function.json = sample\EasyTables\CSharp\QueueToOutItem\function.json
+		sample\EasyTables\CSharp\QueueToOutItem\run.csx = sample\EasyTables\CSharp\QueueToOutItem\run.csx
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "QueueToInItem", "QueueToInItem", "{56AFFB2C-E153-48DD-B29F-F52842DDD1D5}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\QueueToInItem\function.json = sample\EasyTables\CSharp\QueueToInItem\function.json
+		sample\EasyTables\CSharp\QueueToInItem\run.csx = sample\EasyTables\CSharp\QueueToInItem\run.csx
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "QueueToCollector", "QueueToCollector", "{D28B8134-9DCE-4E42-B417-B00559982ECE}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\QueueToCollector\function.json = sample\EasyTables\CSharp\QueueToCollector\function.json
+		sample\EasyTables\CSharp\QueueToCollector\run.csx = sample\EasyTables\CSharp\QueueToCollector\run.csx
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "QueueToTable", "QueueToTable", "{13E0D69C-DEE4-487C-8827-476F6D8BEC69}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\QueueToTable\function.json = sample\EasyTables\CSharp\QueueToTable\function.json
+		sample\EasyTables\CSharp\QueueToTable\run.csx = sample\EasyTables\CSharp\QueueToTable\run.csx
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "QueueToQuery", "QueueToQuery", "{EA96F870-2958-41EC-A2A0-2B712398D678}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\QueueToQuery\function.json = sample\EasyTables\CSharp\QueueToQuery\function.json
+		sample\EasyTables\CSharp\QueueToQuery\run.csx = sample\EasyTables\CSharp\QueueToQuery\run.csx
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "QueueToInItemToOutItem", "QueueToInItemToOutItem", "{EB1C06DC-ED35-4716-AD0F-047B7D64D299}"
```

Comment:
Ok, to confirm, you're going to remove this nested folder and move it to the test scripts folder. I think you can have a couple top level EasyTables samples in this directory following the existing pattern if you want.